test/adapter/deprecation_test.rb

```diff
@@ -0,0 +1,109 @@
+require 'test_helper'
+module ActiveModel
+  class Serializer
+    module Adapter
+      class DeprecationTest < ActiveSupport::TestCase
+        class DeprecatedPostSerializer < ActiveModel::Serializer
+          attribute :body
+        end
+        setup do
+          post = Post.new(id: 1, body: 'Hello')
+          @serializer = DeprecatedPostSerializer.new(post)
+        end
+
+        def test_null_adapter_serialization
```

Comment:
I added these tests to have some kind of regression tests. It's not completely reliable but it's better than nothing.